packages/admin-ui/src/components/NotificationsMain.tsx

```diff
@@ -44,6 +46,17 @@ export default function NotificationsView() {
   ).enabled;
 
   const notifications = [
+    /**
+     * [HOST-CONNECTED-TO-INTERNET]
+     * Tell the user if is connected to internet
+     */
+    {
+      id: "connectedToInternet",
+      linkText: "Check",
+      linkPath: "support/auto-diagnose",
+      body: `**Dappnode host is not connected to internet.** Click **Check** to run the autodiagnose and learn more about it.`,
```

Comment:
I would change the copy from "check" to "navigate" to autodiagnose to check the dappnode health.